ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,187 @@
+    description: Pull request requires no code review (e.g., a sub-repository hash update).
+    color: C6C7F8
+  - name: A3-in_progress
+    description: Pull request is in progress. No review needed at this stage.
+    color: DBC6F8
+  - name: A4-stale
+    description: Pull request did not receive any updates in a long time. No review needed at this stage. Close it.
+    color: F2C5F8
+  - name: A5-backport
+    description: Pull request is already reviewed well in another branch.
+    color: F8C5E7
+  - name: A6-unconfirmed
+    description: Issue might be valid, but it's not yet known.
+    color: "000000"
+  - name: B0-runtime_migration
+    description: PR introduces code that might require downstream chains to run a runtime upgrade.
+    color: c2e0c6
+  - name: B1-database_migration
+    description: PR introduces code that does a one-way migration of the database.
+    color: f4f5f9
+  - name: B2-node_first_update
+    description: This is a runtime change that will require all nodes to be update BEFORE the runtime upgrade.
+    color: e8d7d7
+  - name: B3-introduce_new_host_function
+    description: PR adds new host functions which is not used yet, so no need for upgrade.
+    color: f9f2f2
+  - name: B4-enable_new_host_function
+    description: PR enables new host functions which requires a node release before a runtime upgrade.
+    color: f9f2f2
+  - name: B5-enable_modified_host_function
+    description: INPUT NEEDED FROM @KOUTE
+    color: f9f2f2
+  - name: B6-disable_host_function
+    description: PR disables host functions which requires a node release before a runtime upgrade.
+    color: f9f2f2
+  - name: C0-easy
+    description: Can be fixed primarily by duplicating and adapting code by an intermediate coder.
+    color: eeeeee
+  - name: C1-medium
+    description: Can be fixed by a coder with good Rust knowledge but little knowledge of the codebase.
+    color: eebb00
+  - name: C2-substantial
+    description: Can be fixed by an experienced coder with a working knowledge of the codebase.
+    color: D4C5F9
+  - name: C3-involved
+    description: Can be fixed by an expert coder with good knowledge of the codebase.
+    color: E99695
+  - name: C4-mentor
+    description: An easy task where a mentor is available. Please indicate in the issue who the mentor could be.
+    color: "5319e7"
+  - name: C5-reserved
+    description: This issue should not be picked up by anyone unless approved.
+    color: cdadad
+  - name: D0-nice_to_have_audit
+    description: PR contains trivial changes to logic that should be properly reviewed.
+    color: c5def5
+  - name: D1-needs_audit
+    description: PR contains changes to fund-managing logic that should be properly reviewed and externally audited.
+    color: C3EDC0
+  - name: D2-needs_DevRel_audit
+    description: PR contains changes that are relevant for documentation, should be audited by DevRel before merge.
+    color: C0E7CC
+  - name: D2-audited
```

Comment:
`D2` represent a state and I think it should be removed and replaced by some status in the relevant project

Reply:
yes but the difference here is that these labels trigger a matrix message to sr-labs to review the pr, and AFAIK there's no API for the project fields yet so we will need to continue using the labels for it (correct me if I'm wrong @mordamax)

Reply:
i mean - there are some scripts with these labels https://github.com/search?q=%22-audited%22+org%3Aparitytech+&type=code
but im pretty sure in monorepo they will be reviewed. if we change some labels, we should make somewhere a list of deprecated, and a new ones, so it's easier to traverse the ones that needs to be checked against existing automations

Reply:
Maybe labels aren't the best solution for tracking auditable issues.
An alternative would be to have a specific project for all issues that need to be audited, that would also make it easy to see what's the progress on all those issues. And, at the same time we remove this dependency to labels.
@the-right-joyce @mordamax WDYT?

Reply:
@juangirini currently, as @the-right-joyce mentioned, it's used not only for tracking but for automation of release changelog starting from here https://github.com/paritytech/polkadot/blob/bddd25c8fdabb8f5c540d21f70bf1af96809114e/scripts/ci/github/generate_release_text.rb#L97
to somewhere https://github.com/paritytech/polkadot/blob/bddd25c8fdabb8f5c540d21f70bf1af96809114e/.github/workflows/release-99_bot.yml#L37 or 
https://github.com/paritytech/polkadot/blob/bddd25c8fdabb8f5c540d21f70bf1af96809114e/.github/workflows/release-30_publish-draft-release.yml#L117

though it's true, so many labels might not be the best thing overall

Here we discuss the alternative way to keep changelog up to date and more meaningful with a goal to reduce amount of labels https://github.com/paritytech/release-engineering/issues/165 hopefully it could lead to deprecation of many labels like this. Please join with your thoughts there

Reply:
yes, but what I was asking is: if we don't use labels and instead use the project status/fields for the auditing process we won't be able to trigger a msg to sr-labs in Matrix right? Bc the API of the projects doesn't give back the value of the fields? Or am I recalling this wrong?

Reply:
@juangirini yes I think an own project board for that would make sense. Let me speak to SR labs and see if that would be ok for them as well.

Reply:
Got the confirmation from SR Labs that a project works for them as well and here it is: https://github.com/orgs/paritytech/projects/103
We just need an automation now to add items automatically to the project